restore nRF52 support (#794)
nrfconnect/sdk-sidewalk: doc/includes/include_testing_nordic_dfu.txt

Complete the following steps to test the Device Firmware Update (DFU) on your device:

.. tabs::

   .. group-tab:: nRF52 DKs

      #. Enter the DFU mode by long pressing **Button 1** on your development kit.

>          This action sets the device in the DFU mode, in which only the `Zephyr SMP Server`_ is running and Sidewalk is not operational.

Reviewer: ```suggestion
         This action sets the device to DFU mode, in which only the `Zephyr SMP Server`_ is running and Sidewalk is not operational.
```

         When the application is in the DFU mode, all LEDs flash every 500 ms to signal that the application is waiting for a new image.

      #. To perform a firmware update, follow the Bluetooth testing steps from the `DevZone DFU guide`_.
         When the update starts, the LEDs change the pattern to indicate upload in progress.

>       #. To exit the DFU mode, long press **Button 1** again when device is in DFU mode.

Reviewer: ```suggestion
      #. To exit the DFU mode, long-press **Button 1**.
```

>          The device will set the device in the Sidewalk mode.

Reviewer: ```suggestion
         The device will switch to Sidewalk mode.
```

>          If the update completes successfully, the device will start a new image.

Reviewer: ```suggestion
         If the update completes successfully, the device starts a new image.
```

         After using the DFU mode, the first bootup might take up to 60 seconds.

>          During the image swap the application is silent, meaning that LEDs are not blinking and the console does not output any logs.

Reviewer: ```suggestion
         During the image swap, the application is silent, meaning that LEDs are not blinking and the console does not output any logs.
```

         The application, however, is running, so do not reset the device.

   .. group-tab:: nRF54 DKs

      #. Enter the DFU mode by long pressing **Button 0** on your development kit.

>          This action sets the device in the DFU mode, in which only the `Zephyr SMP Server`_ is running and Sidewalk is not operational.

Reviewer: ```suggestion
         This action sets the device to DFU mode, in which only the `Zephyr SMP Server`_ is running and Sidewalk is not operational.
```

         When the application is in the DFU mode, all LEDs flash every 500 ms to signal that the application is waiting for a new image.

      #. To perform a firmware update, follow the Bluetooth testing steps from the `DevZone DFU guide`_.
         When the update starts, the LEDs change the pattern to indicate upload in progress.

>       #. To exit the DFU mode, long press **Button 0** again when device is in DFU mode.

Reviewer: I think it's redundant
```suggestion
      #. To exit the DFU mode, long-press **Button 0**.
```

>          The device will set the device in the Sidewalk mode.

Reviewer: ```suggestion
         The device will switch to Sidewalk mode.
```

>          If the update completes successfully, the device will start a new image.

Reviewer: ```suggestion
         If the update completes successfully, the device starts a new image.
```
